hedera-node/hedera-app-spi/src/main/java/com/hedera/node/app/spi/accounts/AccountBuilder.java

```diff
@@ -14,7 +14,7 @@
  * limitations under the License.
  */
 
-package com.hedera.node.app.service.token.entity;
+package com.hedera.node.app.spi.accounts;
```

Comment:
Why does `AccountBuilder` need to be moved?

Reply:
Because `Account` has a `copy()` method that returns `AccountBuilder`. Do you think they should be put back to `TokenService` module ?